PetAdoption-iOS/PetAdoption-iOS/Controllers/PetListingViewController.swift

```diff
@@ -175,6 +184,21 @@ class PetListingViewController: UIViewController
         self.present(alertController, animated: true, completion: nil)
     }
     
+    func presentFilterViewController()
+    {
+        guard let filterVC = storyboard?.instantiateViewController(withIdentifier: "filterVC") as? FilterViewController else {fatalError("Error: could not instantiate filterVC")}
```

Comment:
I don't know that I would necessarily want us to fatalError here if the view controller can't be instantiated, since that will just crash the app.  I also don't know that I'd want this to silently return either...  I guess for now this is fine.  At the very least, if this guard does fail, Crashlytics should report the crash and we can investigate further.